met-web/src/components/landing/EngagementTile.tsx

```diff
@@ -76,78 +80,104 @@ const EngagementTile = ({ passedEngagement, engagementId }: EngagementTileProps)
     }
 
     if (!loadedEngagement) {
-        return <MetLabel>{translate('landingPage.tile.error')}</MetLabel>;
+        return <EyebrowText>{translate('landingPage.tile.error')}</EyebrowText>;
     }
 
-    const { name, end_date, start_date, description, banner_url, submission_status } = loadedEngagement;
-    const EngagementDate = `${dayjs(start_date).format(dateFormat)} to ${dayjs(end_date).format(dateFormat)}`;
-
-    const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
-        event.stopPropagation();
-        window.location.href = engagementUrl;
-    };
+    const { name, banner_url } = loadedEngagement;
 
     return (
-        <Card
-            sx={{
-                maxWidth: 345,
-                '&:hover': {
-                    backgroundColor: 'var(--bcds-surface-secondary-hover)',
-                    cursor: 'pointer',
-                },
-            }}
-            onClick={() => {
-                window.location.href = engagementUrl;
-            }}
-        >
-            <When condition={Boolean(banner_url)}>
-                <CardMedia sx={{ height: 140 }} image={banner_url} title={name} />
-            </When>
-            <CardContent>
-                <Box sx={{ minHeight: 200 }}>
-                    <MetHeader4>{name}</MetHeader4>
-                    <MetParagraphOld
-                        sx={{
-                            overflow: 'hidden',
-                            textOverflow: 'ellipsis',
-                            display: '-webkit-box',
-                            WebkitLineClamp: '6',
-                            WebkitBoxOrient: 'vertical',
-                        }}
-                        mt="0.5em"
-                    >
-                        {description}
-                    </MetParagraphOld>
-                </Box>
-                <MetBodyOld bold mt="1em">
-                    {EngagementDate}
-                </MetBodyOld>
-                <Stack direction="row" alignItems={'center'} spacing={1} mt="0.5em">
-                    <MetBodyOld bold>{translate('landingPage.tile.status')}</MetBodyOld>
-                    <EngagementStatusChip submissionStatus={submission_status} />
-                </Stack>
-            </CardContent>
-            <CardActions>
-                <If condition={submission_status === SubmissionStatus.Open}>
-                    <Then>
-                        <Button variant="primary" fullWidth onClick={handleClick as () => void}>
-                            {translate('buttonText.shareYourThoughts')}
-                        </Button>
-                    </Then>
-                </If>
-                <If
-                    condition={
-                        submission_status === SubmissionStatus.Closed || submission_status === SubmissionStatus.Upcoming
-                    }
+        <Link to={engagementUrl} sx={{ textDecoration: 'none' }} tabIndex={-1}>
```

Comment:
Card components don't really have standardized WCAG implementations and I think wrapping it all in an anchor tag is definitely a valid approach! I'd be curious about a couple of things with this approach:

- What is the screen reader saying? How verbose is it?
- I see you used tabIndex="-1" a few times here. Where exactly does a keyboard land when tabbing through?